lib/src/connector/connection.cc

```diff
@@ -210,55 +211,52 @@ void Connection::connect(int max_connect_attempts) {
     } while (try_again);
 
     connection_backoff_ms_ = CONNECTION_BACKOFF_MS;
+    // TODO(ale): deal with locale & plural (PCP-257)
     throw connection_fatal_error {
-        (boost::format(
-            "failed to establish a WebSocket connection after %1% attempt%2%")
-                % idx
-                % (idx > 1 ? "s" : "")).str() };
+        lth_loc::format("failed to establish a WebSocket connection after "
```

Comment:
In this case we'll end up using a special form of translate for pluralization (probably applied to the whole message, rather than just "attempts", /cc @LarissaLane), then pass that to `locale::format` for substitution.